admin_organization.go

```diff
@@ -84,10 +84,10 @@ const (
 type AdminOrganizationListOptions struct {
 	ListOptions
 
-	// A query string used to filter organizations.
+	// Optional: A query string used to filter organizations.
 	// Any organizations with a name or notification email partially matching this value will be returned.
 	Query string `url:"q,omitempty"`
-
+	// Optional:
```

Comment:
This is an interesting and useful parameter that I think we should take better care to document explicitly. Here's an example from WorkspaceReadOptions:

`// Optional: A list of relations to include. See available resources https://www.terraform.io/docs/cloud/api/workspaces.html#available-related-resources`

I think we have good docs around which relations are possible to include, and a link would reinforce the relationship we now have to maintain because they are typed strings.

Reply:
☝️

Reply:
We have a similar comment for the include options in admin_organization.go. You may have not seen it because is in line 75:

```
// A list of relations to include. See available resources
// https://www.terraform.io/docs/cloud/api/admin/organizations.html#available-related-resources
type AdminOrgIncludeOps string
```

So, the issue here is that we have discrepancy where we place that comment and link. Most of the time we do it on top of the type and occasionally we do it the include field definition level, like what you see in workspace.go. I personally, like it a bit more on top of the type because Godoc makes types more visible and will include the typed constants for the include field.